src/data-types/interfaces.ts

```diff
@@ -14,3 +14,19 @@ export interface CurrentTimeDescriptor {
 export interface DeleteStatusUpdateSNSMessage {
   user_id: string;
 }
+
+export interface TransformedResponseFromDynamoDatabase {
```

Comment:
This interface should be named to something that describes the interface better. Maybe something like `AccountStatus`?